proposals/collection-expression-better-conversion.md

```diff
@@ -0,0 +1,125 @@
+# Better conversion from collection expression element
+
+## Summary
+
+Updates to the better conversion rules to be more consistent with `params`, and better handle current ambiguity scenarios. For example, `ReadOnlySpan<string>` vs `ReadOnlySpan<object>` can currently
+cause ambiguities during overload resolution for `[""]`.
+
+## Detailed Design
+
+The following are the better conversion from expression rules. These replace the rules in https://github.com/dotnet/csharplang/blob/main/proposals/csharp-12.0/collection-expressions.md#overload-resolution.
+
+These rules are:
+
+> Given an implicit conversion `C₁` that converts from an expression `E` to a type `T₁`, and an implicit conversion `C₂` that converts from an expression `E` to a type `T₂`, `C₁` is a ***better conversion*** than `C₂` if one of the following holds:
+> **`E` is a *collection expression*, and `C₁` is a ***better collection conversion from expression***, or**
+> * **`E` is not a *collection expression* and one of the following holds:**
+>   * `E` exactly matches `T₁` and `E` does not exactly match `T₂`
+>   * `E` exactly matches both or neither of `T₁` and `T₂`, and `T₁` is a [*better conversion target*](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/expressions.md#11646-better-conversion-target) than `T₂`
+> * `E` is a method group, ...
+
+We add a new definition for ***better collection conversion from expression***, as follows:
+
+Given:
+- `E` is a collection expression with element expressions `[EL₁, EL₂, ..., ELₙ]`
```

Comment:
If `EL` are meant to include spread elements, then we should avoid calling these expressions.
```suggestion
- `E` is a collection expression with elements `[EL₁, EL₂, ..., ELₙ]`
```